api/src/main/resources/workbench.yaml

```diff
@@ -1269,6 +1269,32 @@ definitions:
         type: boolean
       emailVerificationStatus:
         $ref: "#/definitions/EmailVerificationStatus"
+      aboutYou:
+        type: string
+      areaOfResearch:
+        type: string
+      institutionalAffiliations:
+        type: "array"
+        items:
+          $ref: "#/definitions/InstitutionalAffiliation"
+
+  InstitutionalAffiliation:
+    type: object
+    required:
+      - userId
+      - institution
+      - role
+    properties:
+      institutionalAffiliationId:
+        type: integer
+        format: int64
```

Comment:
If you're not going to use the IDs for anything, you don't necessarily need to expose these here.